ecpp

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/python
+
+from __future__ import print_function
+import ConfigParser
+import sys
+import StringIO
+from verify import *
+
+#oneseventwoeight = 1728
+
+try:
+    xrange
+except NameError:
+    xrange = range
+
+# dict with params
+def par_set():
+    global params
+    params = {'s':0,'w':0,'j':0,'t':'0','a':0,'b':0,'q':0,'r':0}
+
+# division function
+def division(n,i):
+    if n % i == 0:
+        raise ValueError("not prime: " + str(n))
+
+with open(sys.argv[1], 'r+') as f:
+    config_buff = f.read()
+    f.close()
+
+config_ini = StringIO.StringIO(config_buff)
+config = ConfigParser.RawConfigParser(allow_no_value=True)
+config.readfp(config_ini)
+sections = config.sections()
+
+par_set()
+
+# combinations
+c_t1 = ['s','w','j','t']
+c_t2 = ['s','w','a','b','t']
+llr_t = ['s','q']
+bls_t = ['s','b']
+
+# prime candidate
+n = config.get('Candidate','N')
+print(n)
+n = int(n.replace("$",""), 16)
+
+print("===============")
+print("Candidate: ")
+print(n)
+print("===============")
+
+
+section = str(1)
+run = True
+while run : 
+    if section in sections:
+        par_list = []
+        options = config.options(section)
+        print("===============")
+        print(section)
+        print("===============")
+        for option in options:
+            print(option + '=' + config.get(section,option))
+            value = config.get(section,option)
+            # if starts with minus
+            params[str(option)] = value.replace("$","")
+            par_list.append(option)
+        print(par_list)
+        print(params)
+# TODO: add some clean function to the end of the section, clean values in params
+        if set(c_t1) == set(par_list):
+            print("start ----> c_t1")
+            s = int(str(params['s']),16)
```

Comment:
why `str()` here, and in following lines?

also nit: there should be space after comma